relayer/src/connection.rs

```diff
@@ -828,7 +828,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Connection<ChainA, ChainB> {
             .map_err(|e| ConnectionError::chain_query(self.dst_chain().id(), e))?;
         let client_msgs = self.build_update_client_on_src(src_client_target_height)?;
 
-        let tm = TrackedMsgs::new(client_msgs, "create connection");
+        let tm = TrackedMsgs::new(client_msgs, "update client on source for ConnectionOpenTry");
```

Comment:
I can't think of a shorter way to formulate this without losing context. I guess we can keep it as it is.